### DISCUSSION TITLE
Application to GitHub Education

### DISCUSSION THREAD

**Discussion by 37si** [community/community#139126] [GitHub Education]
### Hi everyone,

Hi everyone,

I recently applied for GitHub Education, but I haven’t received my student ID card yet. I’m currently in the process of moving houses, and my classes won’t start for another two weeks, so it might take a little longer to get the ID. However, I do have access to my official college email, which I know can sometimes be used as proof of enrollment.

Would it be possible for my application to be reviewed with the email as verification in the meantime? I’d really appreciate any advice or support on this!

Thanks in advance for your help!

Chris

**Comment by davevad93** [comment]
Hi @cwilliam1 ,

The academic email is required, but additional proof is needed to apply successfully,, as stated in the [docs](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-education-for-students/apply-to-github-education-as-a-student)  "***Documents that prove your current student status include a picture of your school ID with current enrollment date, class schedule, transcript, and affiliation or enrollment verification letter.***"

So you can use one of the above documents, for example, in my last application, I uploded a photo of my last exam results at that time and it was accepted.
